### PR TITLE
feat: use configured work days for weekly pace

### LINKED ISSUE #1356: feat: factor weeklyProgressWorkDays into pace calculation
## Summary

The existing `weeklyProgressWorkDays` setting (added in #1096 / PR #1102) segments the weekly progress bar visually, but does not affect the pace calculation itself. Users who work only on weekdays (e.g. 5 days/week) see misleading pace indicators throughout the week:

| Day (5-day user) | 7-day linear expected | Actual usage | Pace verdict |
|---|---|---|---|
| Wednesday | 42.9% | 60% | **17% deficit** |
| Friday | 71.4% | 100% | **29% deficit** |
| Saturday | 85.7% | 100% | 14% deficit |

A user consuming 100% by Friday is using their quota exactly as planned across 5 work days, yet the current model reports a large deficit all week.

## Proposed behavior

When `weeklyProgressWorkDays` is set (e.g. 5), the pace expected curve should distribute 100% across only the configured work days rather than uniformly across 7 calendar days:

| Day (5-day model) | Expected | Actual | Verdict |
|---|---|---|---|
| Wednesday | 60% | 60% | **on pace** |
| Friday | 100% | 100% | **on pace** |
| Saturday | 100% | 100% | on pace |

On non-work days, the expected curve should remain flat (no additional expected usage).

## Implementation notes

`UsagePace.weekly()` currently computes:
```swift
let expected = ((elapsed / duration) * 100).clamped(to: 0...100)
```

This could be replaced with a workday-aware calculation when `workDays` is provided, distributing expected usage only across work-day hours within the window.

The `weeklyProgressWorkDays` value is already available in settings — it just needs to be threaded into the pace calculation path.

## Context

Follow-up to #1096 which added visual segmentation. This completes the feature by making the pace indicator workday-aware.

### PR DESCRIPTION
**Pull request by steipete**
## Summary

- use configured work days for standard weekly pace calculations
- keep historical Codex pacing unchanged when historical data is available
- compute ETA in work time and map it back to wall-clock time
- preserve exhausted pace output before the first configured workday while retaining early-window noise suppression
- use the existing persisted `weeklyProgressWorkDays` value for both weekly markers and fallback pace/ETA; this unified behavior is intentional

Supersedes #1357 because repository CI would not start for the fork head. Preserves @pstanton237's commits and authorship.

Fixes #1356.

## Validation

- exact candidate: `1a60cb5ac26e81153ad00eb12487fc7ac0b13041`
- focused pace, menu-model, historical-authority, settings, and localization suite: 134 tests across 7 suites passed
- timezone stress under `Pacific/Auckland`: 65 tests across 3 suites passed
- full deterministic inventory with capped concurrency: all 427 suites passed (`3,753 + 76 + 132` test executions)
- `make check`: SwiftFormat clean; SwiftLint 0 violations across 1,077 Swift files
- `swift build`
- exact-head autoreview: no accepted/actionable findings
- ad-hoc packaged debug app: strict deep codesign verification passed; executable SHA-256 `6628e502ada679532bb18a14626f138e035d0732198bfc1234225faca882f1e5`
- built-app proof: selected Off, 4 days, and 5 days in Display settings; verified 5 days persisted across restart; exercised the real Codex-backed app and restored the prior absent preference afterward
- the available account exposed no weekly rate-limit event, so the exhausted visible menu path is proved by the focused menu-model regression that renders `Runs out now`
- Public Model Identifier Gate: PASS over the exact diff, tests, workflows, package, retained settings proof, and public text; no new model identifiers or private account payloads are included

### CODE REVIEW

**Comment by steipete** [comment]
Rebased onto current `main` (`8e50e885`) and resolved the stale switcher changelog conflict. Exact head: `22903e3f`.

Autoreview caught and this head fixes one edge case: a fully depleted quota after the final configured workday now remains exhausted instead of reporting that it lasts through the weekend.

Proof:
- pace/history focused suite: 58 tests passed
- added exhausted-weekend regression
- `make check`: passed
- exact-head autoreview: clean, confidence 0.84

Fresh CI is running. Keeping this draft/unmerged for product review of the workday pacing semantics.

**Comment by steipete** [comment]
Rebased onto current `main` (`3fdb549a`), which includes the deterministic `StatusMenuPersistentRefreshTests` fix that was the only failure on the previous head.

Exact head: `c994ca7d`.

Proof:
- `swift test --filter UsagePace`: 70 tests passed.
- `make check`: clean.
- exact-head autoreview: clean, 0.83 confidence.

Fresh CI is running. Keeping this draft and unmerged for the existing product decision on changing the semantics of the persisted work-day setting.

**Comment by steipete** [comment]
Prepared on exact head `999b083de9ba722196b3b1cf4860bcd702b1edb5`.

Proof:
- Rebased onto current `main` at preparation time.
- Focused pace suites: 68 tests across 4 suites passed.
- `make check`
- Codex autoreview: no accepted/actionable findings.
- GitHub CI: both macOS runs, both Linux x64 runs, both Linux ARM64 runs, and GitGuardian passed. The first Linux x64 attempt hit an unrelated `Text file busy` PTY-test flake; rerun passed.

Not merging autonomously: interpreting configured work days as quota-consumption pace changes user-facing product semantics.

**Comment by steipete** [comment]
Prepared on exact head `19a120a90aeee52b37a82d4ad46cf270a711effd`.

Proof:
- focused pace tests passed, including the pre-first-workday and exhausted-weekend regressions
- `make check` passed
- Codex autoreview reported no accepted/actionable findings
- GitHub CI is fully green: macOS, Linux x64, Linux ARM64, and GitGuardian

Keeping this draft and unmerged. The remaining question is product/upgrade semantics: whether the existing persisted work-day display preference should also control quota pace and ETA.

**Comment by steipete** [comment]
Rebased onto current `main` at `1d39e0ca9ec4b53685d6a7c4720df3d9945d68e7`.

Exact head: `3e83eb4c`.

Proof:
- `swift test --filter UsagePace`: 71 tests in 3 suites passed.
- `make check`: SwiftFormat clean; SwiftLint 0 violations.
- exact-head Codex autoreview: no accepted/actionable findings, 0.82 confidence.

Fresh CI is running. Keeping this draft and unmerged for the existing product/upgrade decision: whether the persisted work-day display preference should also control quota pace and ETA.

**Comment by steipete** [comment]
Exact candidate: `166dc07327cc1216abf7633b8946b9aaa82f42af`.

Maintainer decision: accept unified semantics. Existing persisted work-day values intentionally control both weekly markers and fallback pace/ETA. The PR body, localized UI copy, changelog, focused regressions, and built-app proof now state and cover that behavior.

@codex review
@clawsweeper re-review

**Comment by steipete** [comment]
Exact candidate: `15270444a37b3af1b80f4010653fb76c2a9899f8`.

Maintainer decision: accept unified semantics. Existing persisted work-day values intentionally control both weekly markers and fallback pace/ETA. The PR body, localized UI copy, changelog, focused regressions, and built-app proof now state and cover that behavior.

@codex review
@clawsweeper re-review

File: Sources/CodexBar/MenuCardView+ModelHelpers.swift [hunk 1]
```diff
@@ -151,6 +151,18 @@ extension UsageMenuCardView.Model {
...
+    static func standardWeeklyPace(input: Input, window: RateWindow) -> UsagePace? {
+        if let weeklyPace = input.weeklyPace {
+            return weeklyPace
+        }
+        return UsagePace.weekly(
+            window: window,
+            now: input.now,
+            defaultWindowMinutes: 10080,
+            workDays: input.workDaysPerWeek)
+            .flatMap { $0.expectedUsedPercent >= 3 ? $0 : nil }
```
[the remark this answers is not in the document]
Comment (steipete) [reply Sources/CodexBar/MenuCardView+ModelHelpers.swift]: Fixed in `1a60cb5a`. `displayableWeeklyPace(_:)` now preserves `etaSeconds == 0` before applying the normal 3% early-window filter, and every computed weekly fallback uses that helper.

`MenuCardWorkdayPaceTests` reproduces a depleted Sunday before the first configured 5-day workday and asserts the visible menu detail is `100% in deficit` / `Runs out now`. The focused exact-head suite passed 134 tests across 7 suites.

**Comment by steipete** [comment]
Exact candidate: `1a60cb5ac26e81153ad00eb12487fc7ac0b13041`.

The depleted pre-first-workday menu finding is fixed and covered by a focused visible menu-model regression. Exact-head focused/full tests, lint, build, package/signature, built-app settings persistence, autoreview, and the Public Model Identifier Gate all pass.

@codex review
@clawsweeper re-review

**Comment by steipete** [comment]
Decision-ready on exact head `1a60cb5ac26e81153ad00eb12487fc7ac0b13041`.

- CI: macOS lint/build/test, Linux x64, Linux arm64, and GitGuardian are green.
- Reviews: exact-head Codex review is clean; ClawSweeper reports sufficient proof and no contributor-facing repair; all inline review threads are resolved.
- Local proof: focused 134 tests across 7 suites; Auckland timezone stress 65 tests across 3 suites; all 427 suites; lint, build, package, and signature checks passed.
- Built app: Off, 4-day, and 5-day selection plus restart persistence verified; the prior preference was restored. The available account had no weekly event, so the exhausted menu path is covered by the focused visible menu-model regression.
- Public Model Identifier Gate: PASS.

Recommendation: land. Tradeoff: existing persisted 4-day or 5-day marker values now intentionally control fallback weekly pace and ETA, as documented.

**Comment by steipete** [comment]
Landed as `2f92ee97e46edb9d47ee4d2b96d53577f9e1fb21`.

Verification on the exact merged head:

- GitHub CI passed: `lint-build-test`, Linux x64, Linux arm64, and GitGuardian.
- Focused pace/menu/history/localization validation: 134 tests across 7 suites passed.
- Timezone stress under `Pacific/Auckland`: 65 tests across 3 suites passed.
- Full deterministic test inventory: all 427 suites passed.
- `make check` and `swift build` passed.
- Exact-head autoreview reported no accepted/actionable findings.
- Ad-hoc packaged app passed strict deep codesign verification.
- Built-app settings proof covered Off, 4-day, and 5-day selections plus restart persistence; prior settings were restored.
- The exhausted visible menu path was covered by the focused menu-model regression because the available live account had no weekly rate-limit event.

Verification images were not applicable beyond the built-app settings proof already attached in the PR discussion.
